backend/spec/spec_helper.rb

```diff
@@ -54,6 +54,16 @@
   browser_options.args << '--window-size=1920,1080'
   Capybara::Selenium::Driver.new(app, browser: :chrome, options: browser_options)
 end
+Capybara.register_driver :selenium_chrome_headless_docker_friendly do |app|
+  Capybara::Selenium::Driver.load_selenium
```

Comment:
This line should not be required since `selenium/webdriver` is already loaded at line 47. Or there's something else in that method that we could need here? If we need it, it's probably better to call it outside all the `register_driver` definitions?

Reply:
Good catch. We don't need it. Removed.